client.go

```diff
@@ -191,9 +191,9 @@ func NewClient(cfg *Config) (Client, error) {
 	// db string should strictly be OVN_Northbound or OVN_Southbound
 	if db == "" {
 		// default to OVN_Northbound
-		db = dbNB
```

Comment:
i'm prefer to rewrite this to switch, like
```
db := cfg.Db
case  db {
    DBNB,DBSB:
        break
    "":
       db = DBNB
    default:
      error
}
```

Reply:
@vtolstov will handle refactoring in separate PR as this commit is just to rename the vars